### ISSUE TITLE
SQL Server for PostgreSQL

### ISSUE THREAD

**Issue by Magnus512** [mtxr/vscode-sqltools#1110]
Hi everyone, I recently found this amazing plugin to manage my DB, but I'm not able to use language server for Postgres, there is any way to do it?

**Comment by gjsjohnmurray** [comment]
> I'm not able to use language server for Postgres

Please explain what you mean and what you have tried.

**Comment by Magnus512** [comment]
> > I'm not able to use language server for Postgres
> 
> Please explain what you mean and what you have tried.

Hi I'm connecting to postresql Bata Bases, when I try to write some code I don't get table autocompletion for the actual connection, I didn't try anything else because I don't know if I need to enable a feature to use LS

**Comment by gjsjohnmurray** [comment]
I don't think you should need to enable anything specifically. Please confirm what version of the SQLTools and PostgreSQL extensions you are using, and what VS Code version and platform (info about that is in the About option from VS Code's Help menu).

It may also help us if you can paste an animated GIF here which demonstrates what is or isn't happening. The site at https://gifcap.dev/ is one way of creating this. Consider running the `Developer: Toggle Screencast Mode` command before recording. Make sure you don't show any confidential data in the GIF.
